benchmarking/workflow/rules/main.smk

```diff
@@ -272,3 +272,31 @@ rule performance_report:
         "../envs/performance_report.yaml"
     script:
         "../scripts/performance_report.py"
+
+
+rule plot_report:
+    input:
+        performance_report=get_performance_report_output(),
+    output:
+        output_directory="output/reports/mmseqs2",
```

Comment:
I'd recommend passing a file instead of a directory if it's not too much effort. Snakemake can be a bit picky about that (see [this section of the docs](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#directories-as-outputs)), if it's a directory it requires it to be wrapped with the `directory` function. 

I'd also suggest not putting the tool that's being benchmarked in the output name, because if we switch the tool, will have to change here as well.